### PR TITLE
Change as_string to __str__ in nginxparser to make it more pythonic.

### PR DESCRIPTION
**Pull request by lf-**
This change would make the RawNginxDumper more in line with other Python libraries and the standard library.

### CODE REVIEW

File: letsencrypt-nginx/letsencrypt_nginx/nginxparser.py [hunk 1]
```diff
@@ -84,10 +84,14 @@ def __iter__(self, blocks=None, current_indent=0, spacer=' '):
                     else:
                         yield spacer * current_indent + key + spacer + values + ';'
 
-    def as_string(self):
+    def __str__(self):
         """Return the parsed block as a string."""
         return '\n'.join(self) + '\n'
 
+    def as_string(self):
```
Comment (kuba) [inline letsencrypt-nginx/letsencrypt_nginx/nginxparser.py]: Is this still necessary? Can we drop it in whole?

**Comment by kuba** [comment]
Thanks for contribution and welcome to the project, @lf- :) Looks good, but quick grep shows that `as_string` is dead and can be removed, could you do that for me please?

**Comment by lf-** [comment]
@kuba done :smile:

**Comment by bmw** [comment]
Initial travis failure was unrelated to the changes (log can be found [here](https://gist.github.com/bmw/d4233c49744e89158904)).

Thanks for the PR @lf-! LGTM.

**Pull request merged**